tmt/steps/execute/__init__.py

```diff
@@ -166,6 +166,9 @@ def check_shell(self, test):
             data['result'] = {0: 'pass', 1: 'fail'}[test.returncode]
         except KeyError:
             data['result'] = 'error'
+            # Add note about the exceeded duration
+            if test.returncode in [124, -9]:
```

Comment:
could we have these magic constants (124 and -9) defined and documented?

Reply:
Good idea. Added constants for these special return codes.